wandb/sdk/launch/docker.py

```diff
@@ -97,6 +97,8 @@ def build_docker_image(project: _project_spec.Project, name, base_image, api):
         "ENV WANDB_NAME={wandb_name}\n"
         "ENV WANDB_LAUNCH=True\n"
         "ENV WANDB_LAUNCH_CONFIG_PATH={config_path}\n"
+        "ENV WANDB_RESUME=allow \n"
```

Comment:
nit: remove space after allow

Reply:
This whole line should be removed actually. We no longer are reiniting